src/main/java/ru/testtask/controller/AuthController.java

```diff
@@ -0,0 +1,30 @@
+package ru.testtask.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import ru.testtask.config.jwt.JwtProvider;
+import ru.testtask.dto.AuthUserDTO;
+import ru.testtask.model.User;
+import ru.testtask.service.UserService;
+
+@RestController
+@RequestMapping("/api")
+public class AuthController {
+
+    @Autowired
+    private JwtProvider jwtProvider;
+
+    @Autowired
+    private UserService userService;
+
+
+    @PostMapping("/auth")
+    public String auth(@RequestBody AuthUserDTO authUserDTO) {
+        User user = userService.getAuthorizedUser(authUserDTO.getUsername(), authUserDTO.getPassword());
+        return jwtProvider.generateToken(user.getUsername());
```

Comment:
`userService.getAuthorizedUser()` can return `null`, but you don't process it. I suggest to change signature of the method: 
`public Optional<User> getAuthorizedUser(String username, String password) `
If value is empty, return 401 status code to client